src/components/popup/details/purposes/purposes.jsx

```diff
@@ -222,17 +247,40 @@ export default class Purposes extends Component {
 																		   index={index}
 																		   purpose={purpose}
 																		   isActive={purposeIsActive(purpose)}
+																		   isLegitimateInterestActive={isPurposeLegitimateInterestActive(index)}
 																		   isTechnical={false}
 																		   createOnShowVendors={this.createOnShowVendors.bind(this)}
+																		   onToggleLegitInterest={this.handleSelectLegitInterest.bind(this)}
 																		   onToggle={this.createHandleSelectPurpose()}/>)}
 							</div>
+							{specialPurposes && !!specialPurposes.length && <div>
+								<LocalLabel className={style.header} prefix="specialPurposes" localizeKey={`title`}>Special purposes</LocalLabel>
+								{specialPurposes.map((purpose, index) => <Purpose key={index}
+																		   index={index}
+																		   purpose={purpose}
+																		   isActive={false}
+																		   isLegitimateInterestActive={false}
+																		   isTechnical={true}
+																		   specialPurpose={true}
+																		   createOnShowVendors={this.createOnShowVendors.bind(this)}
+																		   onToggleLegitInterest={this.handleSelectLegitInterest.bind(this)}
```

Comment:
Special Purpose” means one of the defined purposes for processing of data, including users’ personal data, by participants in the Framework that are defined in the Policies or the Specifications for which Vendors declare a Legal Basis in the GVL and for which the user is not given choice by a CMP. 
- [ ] special purposes are for informative pupose only